comictaggerlib/taggerwindow.py

```diff
@@ -1655,10 +1681,11 @@ def copy_tags(self) -> None:
                 success_count = 0
                 for prog_idx, ca in enumerate(ca_list, 1):
                     ca_saved = False
-
-                    if ca.has_metadata(src_style) and ca.is_writable():
-                        md = ca.read_metadata(src_style)
-                    else:
+                    md, error = self.overlay_ca_read_style(src_styles, ca)
+                    if error is not None:
+                        failed_list.append(ca.path)
+                        continue
+                    if md.is_empty:
```

Comment:
The `failed_list` is used for writing styles but as a failed read will likely also be a failed write, as per the same with the write error, bail on this archive right away.